addons/survey/views/survey_question_views.xml

```diff
@@ -16,9 +16,7 @@
                     <div class="oe_title" style="width: 100%;">
                         <label for="title" string="Section" attrs="{'invisible': [('is_page', '=', False)]}"/>
                         <label for="title" string="Question" attrs="{'invisible': [('is_page', '=', True)]}"/>
-                        <separator />
-                        <field name="title" colspan="4"/>
-                        <separator />
+                        <field name="title" placeholder="e.g. &quot;What is the...&quot;" colspan="4"/>
```

Comment:
A bit strange but ok, seems it is difficult to have a really dynamic placeholder ^^ .